README.md

```diff
@@ -120,19 +120,22 @@ Here's an overview of the architecture:
 	# (optional) require authentication for viewing content
 	scoold.is_default_space_public = false
 	```
-3. Start Scoold with `java -jar -Dconfig.file=./application.conf scoold-*.jar`
+3. Start Scoold with `java -jar -Dconfig.file=./application.conf scoold-*.jar`.
 4. Open [localhost:8000/signin/register](http://localhost:8000/signin/register) and
-register a new account with same email you put in the configuration
+register a new account with same email you put in the configuration.
 
-If you want to login with a social account, first you *need* to create a developer app with
+To login with a social account, you must create a developer app with
 [Facebook](https://developers.facebook.com),
-[Google](https://console.developers.google.com) or **any other identity provider** that you wish to use.
-This isn't necessary if you're planning to login with LDAP, SAML or with email and password.
+[Google](https://console.developers.google.com) or any other identity provider.
+This is not required when you log in with LDAP, SAML or email/password.
+
+misplaced
 Save the obtained API keys in `application.conf`, as shown below.
 
-> For all identity providers you must whitelist the Para host with the
-appropriate authentication endpoint. For example, for GitHub, the redirect URL would be: `https://paraio.com/github_auth`,
-for OAuth 2 - `https://paraio.com/oauth2_auth` and [so on](http://paraio.org/docs/#029-passwordless).
+> For all identity providers, you must whitelist the Para host with the appropriate authentication endpoint:
+> - GitHub: `https://paraio.com/github_auth`  
+> - OAuth 2: `https://paraio.com/oauth2_auth`  
+> Learn more about [custom authentication](http://paraio.org/docs/#029-passwordless)  
```

Comment:
Do you like the formatting? Is "you must whitelist the Para host with the appropriate authentication endpoint" clear enough? Is it about whitelist authentication? Can you explain?

Reply:
I think this is clear enough for administrators who configure the OAuth2 apps to work with Para - they need to add the Para host URL to the list of allowed redirects on the configuration page of the OAuth2 provider.